pymeasure/instruments/commonBase.py

```diff
@@ -0,0 +1,468 @@
+#
```

Comment:
I would propose to rename file as `common_base.py` instead of using camel case which is normally difficult to handle in Windows.

Reply:
>  camel case which is normally difficult to handle in Windows.

Must be thousands of C++ (etc.) developers tearing their hair out? :D
I think what you mean is difficult do handle if is you end up having `commonBase.py` and `commonbase.py`, but that is down to case-insensitivity of NTFS, not the CamelCase. :-P

I agree anyway with snake_case being a better fit here. :-)

Reply:
Done